WcaOnRails/app/mailers/competitions_mailer.rb

```diff
@@ -20,28 +20,28 @@ def notify_wcat_of_confirmed_competition(confirmer, competition)
     end
   end
 
-  def notify_organizers_of_confirmed_competition(confirmer, competition)
+  def notify_organizer_of_confirmed_competition(confirmer, competition, organizer)
```

Comment:
What about keeping the same function name/arguments and simply doing the loop over organizers in that function?
We even have a check for `organizers.empty?` whereas it could now safely be replaced by a `@competition.organizers.each ...` :)

Reply:
I wanted to align it with the existing notify_organizer methods, for which the controller is the one doing the looping, instead of having two looping in the controller and two looping in the mailer. How does that sound?
Actually forgot to remove the `empty?` check though, I think it's not needed now.

Reply:
Ah I did not notice we were looping in the controller for others too, then it seems fine :)
(I personally think it makes sense to delegate to the mailer how to handle mailing "organizers", but looping in the controller definitely makes sense)

The `empty?` check looks useless indeed :)

Reply:
Right, should be all done now.

Reply:
Right, should be all done now.